### PR TITLE
feat(docs): add plugin example to all serdes

### PR DESCRIPTION
**Pull request by shrutimantri**
Closes https://github.com/kestra-io/docs/issues/1443

### CODE REVIEW

File: src/main/java/io/kestra/plugin/serdes/avro/AvroToIon.java [hunk 1]
```diff
@@ -35,6 +36,25 @@
     title = "Read a provided avro file and convert it to ion serialized data file."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert an avro file to ion format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/avro/AvroToIon.java:42]:
```suggestion
            title = "Convert an Avro file to the Amazon Ion format.",
```

File: src/main/java/io/kestra/plugin/serdes/avro/IonToAvro.java [hunk 2]
```diff
@@ -30,7 +30,7 @@
     examples = {
         @Example(
             full = true,
-            title = "Convert a CSV file to the Parquet format using Avro schema",
+            title = "Convert a csv file to the avro format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/avro/IonToAvro.java:33]:
```suggestion
            title = "Convert a CSV file to the Avro format.",
```

File: src/main/java/io/kestra/plugin/serdes/csv/CsvToIon.java [hunk 3]
```diff
@@ -32,6 +33,25 @@
     title = "Read a csv file and write it to an ion serialized data file."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert a csv file to ion format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/csv/CsvToIon.java:39]:
```suggestion
            title = "Convert a CSV file to the Amazon Ion format.",
```

File: src/main/java/io/kestra/plugin/serdes/csv/IonToCsv.java [hunk 4]
```diff
@@ -35,6 +36,40 @@
     title = "Read an ion serialized data file and write it to a csv file."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Download a csv file, transform it and write the transformed data in csv format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/csv/IonToCsv.java:42]:
```suggestion
            title = "Download a CSV file, transform it in SQL and store the transformed data as a CSV file.",
```

File: src/main/java/io/kestra/plugin/serdes/excel/ExcelToIon.java [hunk 5]
```diff
@@ -35,6 +38,27 @@
 @Schema(
     title = "Read data from Excel into a row-wise ION-serialized format"
 )
+@Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert an excel file to ion format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/excel/ExcelToIon.java:45]:
```suggestion
            title = "Convert an Excel file to the Ion format.",
```

File: src/main/java/io/kestra/plugin/serdes/excel/IonToExcel.java [hunk 6]
```diff
@@ -42,6 +44,31 @@
 @Schema(
     title = "Read an ION-serialized file and transform it to an Excel file"
 )
+@Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Download a csv file, convert it into ion data, and write it in excel format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/excel/IonToExcel.java:51]:
```suggestion
            title = "Download a CSV file and convert it to the Excel file format.",
```

File: src/main/java/io/kestra/plugin/serdes/json/IonToJson.java [hunk 7]
```diff
@@ -42,6 +44,29 @@
     title = "Read an ion serialized data file and write it to a new line delimited json file."
 )
 @Plugin(
+    examples = {
+    @Example(
+        full = true,
+        title = "Download a csv file, convert it into ion data, and write it in json format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/json/IonToJson.java:50]:
```suggestion
        title = "Download a CSV file and convert it to a JSON format.",
```

File: src/main/java/io/kestra/plugin/serdes/parquet/IonToParquet.java [hunk 8]
```diff
@@ -37,6 +38,50 @@
     title = "Read a provided file containing ion serialized data and convert it to parquet."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Read a csv file, transform it and write the transformed data in parquet format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/parquet/IonToParquet.java:44]:
```suggestion
            title = "Read a CSV file, transform it and store the transformed data as a parquet file.",
```

File: src/main/java/io/kestra/plugin/serdes/parquet/ParquetToIon.java [hunk 9]
```diff
@@ -42,6 +43,25 @@
     title = "Read a provided parquet file and convert it to ion serialized data file."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert a parquet file to ion format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/parquet/ParquetToIon.java:49]:
```suggestion
            title = "Convert a parquet file to the Amazon Ion format.",
```

File: src/main/java/io/kestra/plugin/serdes/xml/IonToXml.java [hunk 10]
```diff
@@ -40,6 +42,40 @@
     title = "Read an ion serialized data file and write it to a XML file."
 )
 @Plugin(
+        examples = {
+        @Example(
+            full = true,
+            title = "Read a csv file, transform it and write the transformed data to xml format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/xml/IonToXml.java:48]:
```suggestion
            title = "Read a CSV file, transform it and store the transformed data as an XML file.",
```

File: src/main/java/io/kestra/plugin/serdes/xml/XmlToIon.java [hunk 11]
```diff
@@ -36,6 +37,25 @@
 )
 @Slf4j
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert a xml file to ion format.",
```
Comment (anna-geller) [inline src/main/java/io/kestra/plugin/serdes/xml/XmlToIon.java:43]:
```suggestion
            title = "Convert an XML file to the Amazon Ion format.",
```

**Review by anna-geller — CHANGES_REQUESTED** [review]
minor adjustments

File: src/main/java/io/kestra/plugin/serdes/json/JsonToIon.java [hunk 12]
```diff
@@ -39,6 +41,25 @@
     title = "Read a json file and write it to an ion serialized data file."
 )
 @Plugin(
+    examples = {
+        @Example(
+            full = true,
+            title = "Convert a json file to ion format.",
```
Comment (shrutimantri) [inline src/main/java/io/kestra/plugin/serdes/json/JsonToIon.java:47]:
```suggestion
            title = "Convert a JSON file to the Amazon Ion format.",
```

**Pull request merged**